pkg/kubelet/util/podConverter/podConverter.go

```diff
@@ -126,14 +126,15 @@ func ConvertVmPodToContainerPod(pod *v1.Pod) *v1.Pod {
 	// for now, just ensure resources were converted along with other essentials
 	cpod.Spec.Containers = []v1.Container{
 		{
-			Name:            pod.Spec.VirtualMachine.Name,
-			Image:           pod.Spec.VirtualMachine.Image,
-			ImagePullPolicy: pod.Spec.VirtualMachine.ImagePullPolicy,
-			VolumeMounts:    pod.Spec.VirtualMachine.VolumeMounts,
-			VolumeDevices:   pod.Spec.VirtualMachine.VolumeDevices,
-			TTY:             true,
-			Stdin:           true,
-			Resources:       pod.Spec.VirtualMachine.Resources,
+			Name:               pod.Spec.VirtualMachine.Name,
```

Comment:
the vcpu annotation needs to be set as well, in line 115 above.

Reply:
it is being set in 116, i have not changed that by adding resourcesAllocated. when pod is resized, new vcpu count should be set. what specifically did i miss here?

Reply:
indeed. my bad.